tests/scripts/keadm_e2e.sh

```diff
@@ -134,4 +123,4 @@ echo -e "\nStarting kubeedge..."
 start_kubeedge
 
 echo -e "\nRunning test..."
-run_test
+util::run_keadm_test
```

Comment:
There appears to be differences between run_test and util::run_keadm_test in keadm_e2e.sh.
If the results of the running are constant. Please get remove of the run_test function.

Reply:
Sorry for missing this. After checking, the results are constant.